spring-boot/admin/src/main/java/eu/coatrack/admin/controllers/AdminController.java

```diff
@@ -647,4 +701,11 @@ private ServiceApi loadTestServiceForConsumerWizard() {
         return testServiceApi;
     }
 
+    @RequestMapping(value = "/gatewayhealthmonitor/notification", method = POST)
+    @ResponseBody
+    public void updateNotificationStatusOnGatewayHealthMonitor (@RequestParam String proxyId) {
+        Proxy proxy = proxyRepository.findById(proxyId);
+        proxy.setMonitoringEnabled(!proxy.isMonitoringEnabled());
```

Comment:
Ok, it might be a good idea, I will apply it.
I defined initially, that based on the previous state, this method would just change the state, so if for example the monitoring was true, when the user clicked on the switch, this state would change to false.
However, as you stated before, there might be some out of sync problem between the GUI and database, and this method could generate issues.

Reply:
Done